shapeworks_cloud/core/serializers.py

```diff
@@ -48,6 +60,7 @@ class Meta:
 
 class CachedAnalysisReadSerializer(serializers.ModelSerializer):
     modes = CachedAnalysisModeReadSerializer(many=True)
+    groups = CachedAnalysisGroupReadSerializer(many=True)
```

Comment:
Just use `CachedAnalysisGroupSerializer` here. The reason `CachedAnalysisMode` has a separare "Read" serializer is because we wanted to add an extra field in the case of a GET request. We don't do that with `CachedAnalysisGroup`.